Gemfile

```diff
@@ -102,6 +102,7 @@ group :test do
   gem "capybara-screenshot"
   gem "launchy"
   gem 'magic_test'
+  gem "orderly", "~> 0.1"
```

Comment:
Nice, never knew about this gem. Can you please add a comment above it to explain why it was added? I realize no other gem in this file has that but I've been reading that it is hard to tell what gems are for. That is why, a small comment might be good to explain why it is being added.

Also, since this is only going to be used in the testing environment can we move it there?

Reply:
> Also, since this is only going to be used in the testing environment can we move it there?

It's nested under `group :test`. Is there something else you mean by moving it to the test environment?